client/blocks/comment-detail/comment-detail-header.jsx

```diff
@@ -36,6 +39,8 @@ export const CommentDetailHeader = ( {
 	isEditMode,
 	isExpanded,
 	postTitle,
+	site,
```

Comment:
We don't seem to be passing `site` through from `CommentDetail`.

Reply:
great catch! Thanks. Fixed on 51167355aec01d71122f16037d210cb6b1438dba.